Detect old storage configuration earlier in the playbook

ballot-scality opened the pull request:
The old storage configuration is now detected in the preflight checks

Fixes: #178

ballot-scality committed:
Move earlier the detection of old storage configuration

The old storage configuration is now detected in the preflight checks

Issue #178

NicolasT commented on roles/preflight_checks/action_plugins/validate_inventory.py:
```diff
@@ -68,6 +68,26 @@ def check_no_duplicate_addresses(task_vars):
+            "You are still having the old storage configuration for {host}. "
+            "A breaking change was introduced in MetalK8s 0.2.0 "
+            "and the default LVM Volume Group has been changed "
+            "from 'kubevg' to {metalk8s_lvm_default_vg}."
```
Ah, cool to know templating works here.

NicolasT reviewed (dismissed):
Didn't test, but code-wise LGTM

NicolasT committed:
Merge branch 'development/1.0' into preflight_check_storage

ballot-scality committed:
Display the default VG name when asserting

the default VG name was not correctly displayed.
We have to rely on a static string for the default value if the variable
metalk8s_lvm_default_vg is not defined because default variables of
roles are not part of task_vars dictionary.

Issue #178

NicolasT reviewed (approved):
Didn't test, but code-wise LGTM
